feature-libs/organization/account-summary/components/details/document/filter/account-summary-document-filter.module.ts

```diff
@@ -0,0 +1,28 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { I18nModule } from '@spartacus/core';
+import { AccountSummaryDocumentFilterComponent } from './account-summary-document-filter.component';
+import { NgSelectModule } from '@ng-select/ng-select';
+import { ReactiveFormsModule } from '@angular/forms';
+import {
+  DatePickerModule,
+  FormErrorsModule,
+  NgSelectA11yModule,
+} from '@spartacus/storefront';
+
+@NgModule({
+  declarations: [AccountSummaryDocumentFilterComponent],
+  imports: [
+    CommonModule,
+    RouterModule,
+    I18nModule,
+    NgSelectModule,
+    NgSelectA11yModule,
```

Comment:
RouterModule and NgSelectA11yModule are used?

Reply:
RouterModule can and has been removed, but NgSelectA11yModule is for Accessibility and is required in order to use [cxNgSelectA11y] on ng-select